Week2/6_arry.js

```diff
@@ -0,0 +1,8 @@
+let zoo=[];
+console.log(zoo);
+console.log("Animals that I saw in the Zoo");
+zoo=['Lione', 'Tiger', 'Giraffe', 'Elephant'];
+console.log(zoo);
+zoo[4]=('baby pig');
```

Comment:
Technically this works fine! But it's better to use [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push)

Reply:
Done :+1: